lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1542,3 +1557,26 @@ List<int> longestIncreasingSubsequence(List<int> list) {
   }
   return seq;
 }
+
+/// States that a [ui.SemanticsNode] can have.
+///
+/// SemanticsNodes can be in three distinct states (enabled, disabled,
+/// no opinion).
+enum EnabledState {
+  /// [ui.SemanticsFlag.hasEnabledState] not set.
```

Comment:
Style-Nit Here and below: sentences should start with an upper-case character.

Reply:
done. thanks!